torch/_meta_registrations.py

```diff
@@ -3575,8 +3599,8 @@ def meta_binop_inplace_alpha(self, other, alpha=1):
 
 @register_meta([aten.round.default, aten.round.decimals])
 def meta_round(self, **kwargs):
-    return _elementwise_meta(
-        self, type_promotion=ELEMENTWISE_PRIM_TYPE_PROMOTION_KIND.DEFAULT
+    return elementwise_meta(
+        self, type_promotion=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT
     )
```

Comment:
For unary ops, was prims meta not good enough?

Reply:
I guess so, but I figured it'd be good to have just one source of truth, rather than using a private function from a different module. Everything is a no-op when you have just one tensor, so that should be alright.